modules/vaos/spec/services/jwt_spec.rb

```diff
@@ -5,14 +5,58 @@
 describe VAOS::JWT do
   subject { VAOS::JWT.new(user) }
 
-  let(:user) { build(:user, :mhv) }
+  let(:user) { build(:user, :vaos) }
+  let(:rsa_private) { OpenSSL::PKey::RSA.generate(4096) }
```

Comment:
Any way to avoid calling out to generate a new key? Probably not a big deal, just wondering if there's a significant performance cost here.

Reply:
Oh I don't think any of these are actually calling out. They should all be using a VCR cassette. And if you look in UserService once the JWT is returned it is cached for 15 minutes.